components/datetime/src/pattern/runtime/plural.rs

```diff
@@ -10,7 +10,7 @@ use crate::{
 };
 use either::Either;
 use icu_plurals::{PluralCategory, PluralRules};
-use icu_provider::{yoke, zerofrom};
+use icu_provider::prelude::*;
```

Comment:
Note that since you no longer export `yoke` and `zerofrom` from the crate, this is technically a breaking change, but these re-exports were always doc(hidden), so it's probably fine

Reply:
Yep, we've not been treating `doc(hidden)` as semver relevant.